lib/CandidateInfo.js

```diff
@@ -175,4 +180,14 @@ CandidateInfo.expand = function(plain)
 	);
 };
 
+/**
+ * Expands a plain JSON object containing an CandidateInfo or a CandidateInfo and clone it
+ * @param {CandidateInfoPlain | CandidateInfo} plain JSON object or CandidateInfo
+ * @returns {CandidateInfo} Cloned Candidate info
+ */
+CandidateInfo.clone = function(plain)
+{
+	return CandidateInfo.expand(plain).clone();

```

Comment:
nit, this is slightly less efficient than our current pattern:

~~~ js
thing = thing.constructor.name === "MediaInfo" ? thing.clone() : MediaInfo.expand(thing);
~~~

because if passed a plain, we'll expand AND clone